packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -225,9 +225,11 @@ module.exports = React.createClass({
 
   renderComponent() {
     return _.reduce(this.props.proxies, (accumulator, proxy) =>
-      React.createElement(proxy, _.assign({}, this.props, {
-        children: React.Children.only(accumulator),
-      })), this.loadChild('preview'));
+      React.createElement(proxy, _.assign({},
+        this.constructor.getSelectedFixtureContents(this.props), {
```

Comment:
I've missed this the 1st time, but this needs further tweaking. We should be consistent in what we send to proxies vs what we send to the preview component. [THIS](https://github.com/skidding/react-cosmos/blob/42aaa56eafc738efe6a2daa5942cf8bcea6b8b94/packages/react-component-playground/src/components/component-playground.jsx#L132-L144) is what we send to the preview component. We don't need all of those keys, but we need `this.state.fixtureUnserializableProps` merged with `this.state.fixtureContents` to get the latest fixture contents (possibly updated by the user via the shinny editor 😉 ). We might need the component class as well in the future, in case proxies want to, say, display meta data of components on top of previewing them. 

Proposition: Create a method that returns the current fixture contents:
```js
return _.assign({
  component: this.constructor.getSelectedComponentClass(this.props)
}, this.state.fixtureUnserializableProps, this.state.fixtureContents);
```
Then use it as proxy props and inside `preview()` children function, where you add `key`, `ref` and omit `state`.